packages/frontend/src/renderer/components/Sidebar/IdentityPanel/IdentityPanel.tsx

```diff
@@ -48,7 +48,7 @@ export const IdentityPanel: React.FC<IdentityPanelProps> = ({
   const classes = useStyles({})
 
   let communityName = ''
-  if (currentCommunity) {
+  if (currentCommunity?.name.length >= 1) {
```

Comment:
I think it's enough to just check `currentCommunity?.name`

Reply:
Is it? I'd rather stayed with this solution as it doesn't make the code much more complicated